### PR TITLE
Migrate to Github Actions

### PR DESCRIPTION
**Pull request by webframp**
Travis has been slowly dying IMO and slowing down work on this cookbook,
with long delays for jobs to finish on PRs.

Community cookbook repos such as those maintained by sous-chefs:

* https://github.com/sous-chefs

are migrating to using GH actions. Making this repo use the same basic
CI configuration will benefit from following community patterns and be
less surprising to possible contributors

Note that the sous-chefs example use the `delivery` cli to run some
jobs. I have followed that pattern by using the `remote_file` method to
use the same community configuration for which jobs are run by this cli

Remove foodcritic from PR template, it's time to say goodbye.

see:
* https://blog.chef.io/goodbye-foodcritic/

Fixup .md files to pass markdownlint

----

## Pull Request Checklist

**Is this in reference to an existing issue?**

No

#### General

- [X] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [X] Update README with any necessary configuration snippets

- [x] Cookstyle (rubocop) passes

- [x] Rspec (unit tests) passes

- [x] Inspec (integration tests) passes

#### New Features

- [ ] Added a [Testing Artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts) as either an automated test or a manual artifact on the PR.

- [ ] Added documentation for it to the `README.md`

#### Purpose

Overhaul current ci configuration for running kitchen ci

#### Known Compatibility Issues

Unknown

### CODE REVIEW

**Comment by webframp** [comment]
Reference:
* https://docs.github.com/en/free-pro-team@latest/actions

**Comment by webframp** [comment]
@majormoses I think I caught everything cutting over, but appreciate the extra eyes or other feedback on this change.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+"on":
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  delivery:
+    runs-on: ubuntu-latest
```
Comment (majormoses) [inline .github/workflows/ci.yml:12]: Looking at https://docs.github.com/en/free-pro-team@latest/actions/reference/specifications-for-github-hosted-runners 
> Note: The Ubuntu 20.04 virtual environment is currently provided as a preview only. The `ubuntu-latest` YAML workflow label still uses the Ubuntu 18.04 virtual environment.

Should we explicitly use newer?

File: TESTING.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+### Integration Testing
+
 Integration testing with [Test Kitchen](https://docs.chef.io/kitchen.html) can also be done using the delivery cli. To execute all stages of testing with test kitchen you can run either `delivery local acceptance` or `kitchen test`
 
 Test Kitchen is configured to use vagrant by default and uses [inspec](https://www.inspec.io/) to verify.
 
 A configuration is also provided to use [kitchen-dokken](https://github.com/someara/kitchen-dokken) for testing. To use dokken instead of vagrant docker must be installed.
 
-```
+```text
```
Comment (majormoses) [inline TESTING.md:27]: Not that it gives that much better highlighting :shrug: 
```suggestion
```sh
```

**Review by majormoses — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (webframp) [reply .github/workflows/ci.yml:12]: I have no preference for now.

**Comment by webframp** [comment]
I also see this as a path to enabling better Windows platform test coverage but wanted to leave that for it's own PR.

**Pull request merged**